main.go

```diff
@@ -48,6 +48,11 @@ func main() {
 		glog.Errorf("Failed to create ClusterInfo: %v", err)
 	}
 
+	err = setTsiNamespace("/etc/webhook/tsi-namespace")
+	if err != nil {
+		glog.Errorf("Failed to load the TSI Namespace: %v", err)
+	}
+
```

Comment:
Let's put in some a check or something that prevents the server from running if the above checks are not successful. Some of them are slightly outdated. So we see if we still need them.

Reply:
Done